test/formal/opcodes.py

```diff
@@ -58,5 +58,10 @@ def SAR(x, y):
 	return y >> x
 
 def BYTE(i, x):
+	bitInt = (BV2Int(i) + 1) * 8
 	bit = (i + 1) * 8
-	return If(UGT(bit, x.size()), BitVecVal(0, x.size()), (LShR(x, (x.size() - bit))) & 0xff)
+	return If(
+		bitInt > x.size(),
```

Comment:
Would
```suggestion
		i > x.size() / 8 - 1,
```
also work?

Reply:
(I guess we can assume ``x.size()`` to be a multiple of 8 and greater than 1 and then it's the same without any casting or non-constant arithmetic?)